app/views/pieces/show.html.haml

```diff
@@ -1,7 +1,7 @@
 - provide(:title, 'Show')
-
-.embed-container
-  %iframe{width: "560", height: "315", src: "http://www.youtube.com/embed/#{@youtube_url}"}
+.video-container
+  .embed-container
+    %iframe{width: "560", height: "315", src: "http://www.youtube.com/embed/#{@youtube_url}"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Line is too long. [93/80]